src/Commands/AdminCommands/SendtochannelCommand.php

```diff
@@ -81,7 +81,7 @@ public function execute()
         $notes = &$this->conversation->notes;
         !is_array($notes) && $notes = [];
 
-        $channels = (array)$this->getConfig('your_channel');
+        $channels = (array) $this->getConfig('your_channel');
```

Comment:
We should decide on a specific format and keep it that way, otherwise these little things will be overwritten all the time by each other.
At the moment there is some mix of this, some places are one way, others another.
I'll set up an issue for us to discuss this briefly, as it's not such a big deal...